### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by alexkarle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# aql-editor-backend-spring-boot-starter
\ No newline at end of file
+# Spring boot starter for the AQL editor backend
+
+Added as dependency the starter will expose the following REST endpoints:
+
+- /aqleditor/v1/aql
+- /aqleditor/v1/containment
+- /aqleditor/v1/template
```
Comment (stefanspiska) [inline README.md:7]: you changed the path. which means the backend postman Collection no longer works
see https://github.com/ehrbase/aql-editor-backend/pull/10#pullrequestreview-571949299

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        URL website = new URL(urlString);
+        ReadableByteChannel rbc;
+        rbc = Channels.newChannel(website.openStream());
+        FileOutputStream fos = new FileOutputStream(destination);
+        fos.getChannel().transferFrom(rbc, 0, Long.MAX_VALUE);
+        fos.close();
+        rbc.close();
+    }
+
+}
```
Comment (stefanspiska) [inline .mvn/wrapper/MavenWrapperDownloader.java]: Way is maven wrapper in the project?

Comment (alexkarle) [reply .mvn/wrapper/MavenWrapperDownloader.java]: copy, paste. removed

File: README.md (hunk 1 above)
Comment (alexkarle) [reply README.md:7]: changed in aql-editor-backend

**Review by danielatomoiaga — APPROVED** [review]

**Pull request merged**